data/json/items/comestibles.json

```diff
@@ -2620,8 +2620,8 @@
         "price" : 500,
         "material" : "flesh",
         "volume" : 1,
-        "charges" : 1,
-        "stack_size" : 2,
+        "stack_size" : 4,
```

Comment:
Did you test it? The way I'm reading it, it will be 1 volume for up to 4 charges, then 2 volume for up to 8 charges etc. which would be 2x as space-efficient as cooking oil.